docs: Document NOTE_BODY_TAB_LABEL configuration option (#1267)
canvas-medical/documentation: collections/_sdk/handlers/applications.md

| `IDENTIFIER` | A unique key for the application (recommended format: `plugin_name__app_name`) |
| `PRIORITY`   | Controls tab order — lower values appear first. Defaults to `0`                |

> > **Tip:** If your Note Application is named "Note", it may cause confusion with the built-in Note tab. Administrators can rename the built-in tab using the `NOTE_BODY_TAB_LABEL` Constance Config, found in the General Section of your instance Settings, to avoid duplication.

Reviewer: ```suggestion
> **Tip:** If your Note Application is named "Note", it may cause confusion with the built-in Note tab. Users can rename the built-in tab by updating the Constance Config setting `NOTE_BODY_TAB_LABEL` in your instance Settings, to avoid duplication.
```
